lightning/src/ln/functional_test_utils.rs

```diff
@@ -1566,19 +1566,22 @@ macro_rules! get_payment_preimage_hash {
 	};
 }
 
+pub fn get_route(send_node: &Node, payment_params: &PaymentParameters, recv_value: u64, final_cltv_expiry_delta: u32) -> Result<Route, msgs::LightningError> {
+	let scorer = TestScorer::with_penalty(0);
+	let keys_manager = TestKeysInterface::new(&[0u8; 32], bitcoin::network::constants::Network::Testnet);
+	let random_seed_bytes = keys_manager.get_secure_random_bytes();
+	router::get_route(
+		&send_node.node.get_our_node_id(), payment_params, &send_node.network_graph.read_only(),
+		Some(&send_node.node.list_usable_channels().iter().collect::<Vec<_>>()),
+		recv_value, final_cltv_expiry_delta, send_node.logger, &scorer, &random_seed_bytes
+	)
+}
+
 #[macro_export]
 macro_rules! get_route {
```

Comment:
Should we continue adding the deprecation warning / note here and for the following macros?